packages/frontend/src/components/formElements/DatePicker/utils/index.ts

```diff
@@ -1,4 +1,11 @@
-export const getFormatDate = (date: Date | string) => {
+export const getFormatDate = (date: Date | string, separator?: "/" | "." | "-") => {
+  if(typeof date ==='string' && separator){
```

Comment:
It would be better not to simplify this part, because some parts are dedicated to the DatePickers, and it displays some errors when trying to pick a date.
![image](https://github.com/user-attachments/assets/fcdffa68-2617-4711-9159-ba121b9e5967)

Reply:
Let's rewrite it like this then, it won't break Datepicker:
``` typescript
export const getFormatDate = (date: Date | string, separator: '/' | '.' | '-' = '-') => {
  const formattedDate = typeof date === 'string' ? new Date(date) : date;

  const day = String(formattedDate.getDate()).padStart(2, '0');
  const month = String(formattedDate.getMonth() + 1).padStart(2, '0');
  const year = formattedDate.getFullYear();

  return (typeof date === 'string' ? [day, month, year] : [year, month, day]).join(separator);
};
```

Reply:
Done. Working as should be.